ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-stable-4.17-upgrade-from-stable-4.16.yaml

```diff
@@ -59,21 +59,29 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: aws-ipi-ovn-ipsec-amd-mixarch-day2-f14
-  cron: 42 5 14,28 * *
+- as: aws-ipi-ovn-ipsec-amd-mixarch-day2-f28
+  cron: 52 5 15 * *
   steps:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
     test:
-    - chain: openshift-upgrade-qe-sanity-heterogeneous
+    - ref: ipi-install-heterogeneous
+    - chain: openshift-upgrade-qe-sanity
```

Comment:
Seem like the job is changed since my last [comment](https://github.com/openshift/release/pull/54740#discussion_r1690951824), so the comment is not valid any more. `chain: openshift-upgrade-qe-sanity-heterogeneous` is the right way (upgrading cluster, then change the cluster to a heterogeneous cluster).